### PR TITLE
add s3 example with managed resource

### PR DESCRIPTION
**Pull request by csantanapr**
### What does this PR do?

Add example of s3 managed resource

### Motivation

I was looking for a simple example of s3

### More

- [x] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)

- [ ] Yes, I have added a new example under [examples](https://github.com/aws-samples/crossplane-aws-blueprints/tree/main/examples) to support my PR

- [ ] Yes, I have updated the [docs](https://github.com/aws-samples/crossplane-aws-blueprints/tree/main/doc) for this feature

- [ ] Yes, I have linked to an issue or feature request (applicable to PRs that solves a bug or a feature request)

**Note**:
 - Not all the PRs require examples and docs
 - We prefer small, well tested pull requests. Please ensure your pull requests are self-contained, and commits are squashed

### For Moderators
- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: examples/aws-provider/managed-resources/s3.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: s3.aws.crossplane.io/v1beta1
+kind: Bucket
+metadata:
+  name: test-channel-bucket
```
Comment (nimakaviani) [inline examples/aws-provider/managed-resources/s3.yaml:4]: can we have the name be consistent with what we have [here](https://github.com/awslabs/crossplane-on-eks/blob/d4d5a45b95fb841538ce9bd5a28fbc5f6dc5206d/examples/aws-provider/managed-resources/vpc.yaml#L9)

maybe `aws-provider-s3-bucket`?

**Review by nimakaviani — CHANGES_REQUESTED** [review]

File: examples/aws-provider/managed-resources/s3.yaml (hunk 1 above)
Comment (csantanapr) [reply examples/aws-provider/managed-resources/s3.yaml]: sounds good 👍

**Comment by csantanapr** [comment]
@nimakaviani PTAL when you have a chance, I addressed the review feedback

**Review by nimakaviani — APPROVED** [review]
LGTM

**Pull request merged**